app/controllers/writable_controller.rb

```diff
@@ -209,7 +209,7 @@ def process_npc(writable, permitted_character_params)
     )
   end
 
-  def permitted_params(param_hash=nil)
+  def permitted_params(param_hash=nil, extra_params=[])
```

Comment:
What's extra_params doing?

Reply:
When I'm fetching the data of the multi replies JSON from the hidden field on the page, the very first reply might have a reply ID in case I'm editing a reply (rather than posting new ones), and so I pass :id as an extra permitted param there

Reply:
Aha, that makes sense.